Content.Shared/Anomaly/SharedAnomaly.cs

```diff
@@ -33,6 +33,7 @@ public enum AnomalousParticleType : byte
     Delta,
     Epsilon,
     Zeta,
+    Sigma,
```

Comment:
```


                                                                        
                                 %%##%%*@%%                             
                            *#+*##*@@@@@@@@@@@@                         
                         %%*#@%%@@@@%%@@@@@@@@@@@                       
                       %**%*%#@%@%%@%%@@@@@@@@@@@@@                     
                     +#%**#*#%%%%%%%%@%%@@@@@@@@@@@@                    
                     +@*##*==++******######%%%@@@@@@@                   
                    *#%#*====+************###%%%@@@@@@                  
                   %@%@==--:-++*********####%#%%%@@@@@                  
                  *#%**=-::-=++*******######%%%#%@@@@@%                 
                   *%%+=-:-==++********######%%%%%@@@@@                 
                   *%%+=-:--===+*******#####%%%%%%@@@@@                 
                   #*%+=-:===+****##*######%##%%%%@@@@%                 
                   *@*+==*%%%@%%%#####%@%@@@@%%%#%%@@@@                 
                   *%%+:=*%%@@@@@#**%%%@@@@@@%%%%%%@@@@%                
                   *#+=:=*==*##%*=:*#%%@@%%%%%%%##%@@@%                 
                    +:=:-++++***=--*#%%%%%#%%###%#%@%%#                 
                    ***-:=+***==:::+#%%%%%%#####%#%%%%                  
                    *=+=++****+=-:=*#%%%%%%%##%%%##%%*                  
                     =-++*******===*#%%%%%%%%%%%%#%%#                   
                      #=****+++++++%%%%%%%%%%%%%%#%%                    
                       *+*+==-+=+=*%#%%%%%%%%%%%%%@%                    
                        =+===++==+#**##%%%%%%%%%%@@                     
                         ++==*%%%#%@%%%@%%%%%%%%%@@                     
                          ===+***#%%%%%%%%%%%%%%%@@%                    
                          =+=+***%%%%%%%%%%%%%%%%@                      
                          *=+++++**####%%%%%%%%%%*                      
                          ==+*+***#%%%%%%%%%%%%%@*%                     
                         %:==+**+%%@@%%%%%%%%%%@%**@@                   
                        %%-=#+****#%%%%%%%%%%%***##@@@@                 
                       %@#:*--##**%%%%%%%%%@@%%%%#%@@@@@@@              
                    %%%%@@-:+****%%%%%%%%%+###%#*%@@@@@@@@@@@@          
               #%%%@%%%%@@-=::-==*%@%%%%%%#**+**%%@@@@@@@@@@@@@@@@%     
            %@@@@@%%%%%#@@%=***#*%@%%%@%*%%%%%%%#@@@@@@@@@@@@@@@@@@@@@@%
      %%%%@@@@@@@@%@%%%%@@@=-:#@##%%%*@@@@@#***#@@@@@@@@@@@@@@@@@@@@@@@@
%%%%%@@@@@@@@@@@@%@@%%%%@@@@@@@@@+#*%%@@@@@@#@%%@@@@@@@@@@@@@@@@@@@@@@@@
@%%@@@@@@@@@@@@@%@@%%%%@%@@@%*%@@%%*%%@@@@@*%@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@%%@@@@@@@**%@%%%%%@@@@%#@%@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@%@@@@%@@@**%@@*@+@@@%@%%%%@@@@@@@@@@@@@@@@@@@@@@@@@@
```

Reply:
SIGMA

I've studied game design documents and I can tell you it's pretty "seriously stupid."